src/MainWindow.cpp

```diff
@@ -840,7 +840,8 @@ void MainWindow::doubleClickTable(const QModelIndex& index)
     editDock->allowEditing(allowEditing);
 
     // Load the current value into the edit dock
-    editDock->loadText(index.data(Qt::EditRole).toByteArray(), index.row(), index.column());
+    editDock->loadDataFromCell(index.data(Qt::EditRole).toByteArray(),
+                               index.row(), index.column());
```

Comment:
I personally hate this 80 char limit 😆  But I do understand that others are using the terminal for editing. Nonetheless, it feels an odd, obsolete rule IMHO. We have 5K displays, fancy apps, and more than 80 chars per line is causing problems?

Not trying to be rude or offensive to anyone!

Anyways, this can be achieved by extracting long parameters like the first one in variables, which have these advantages:

- easier debugging
- better understanding of the usage through the name of it (e.g. `selectedTableCellData` instead of `index.data(Qt::EditRole).toByteArray()`)
- easier understanding of the call, by having fewer characters on the same line